contracts/nft/contracts/evm/UniversalNFTCore.sol

```diff
@@ -163,6 +163,11 @@ abstract contract UniversalNFTCore is
             revert TransferToZetaChainRequiresNoGas();
         }
 
+        address owner = _requireOwned(tokenId);
```

Comment:
adding unit tests for this would be nice

also are there unit tests for this repo? i suggest to have unit tests for all important smart contracts, maybe we can open issue to add them

Reply:
Ok! Will move tests into this repo in a separate PR 👍

Reply:
https://github.com/zeta-chain/standard-contracts/issues/73